server/chat-commands/moderation.ts

```diff
@@ -605,9 +626,9 @@ exports.commands = {
 
```

Comment:
9 lines up from here, you should declare userid as having type `ID`.

Like:

```
let name;
let userid: ID;
```

Reply:
You clicked "resolve" but you never actually did this thing. :[

Reply:
i did it locally, just am gonna mass-push everything